### PR TITLE
Line-break the bubbles and sort them sensibly.

### PR DESCRIPTION
**Pull request by seancribbs**

### CODE REVIEW

**Comment by seancribbs** [comment]
@joedevivo `<br />` would add a break even if there is only one bubble. bad juju

File: assets/javascripts/app/models.js [hunk 1]
```diff
@@ -103,6 +103,20 @@ GiddyUp.Log = DS.Model.extend({
   }.property()
 });
 
+var propertyComparator = function() {
```
Comment (cmeiklejohn) [inline assets/javascripts/app/models.js]: Maybe just put this in the GiddyUp namespace?

**Comment by cmeiklejohn** [comment]
Otherwise, :+1:

File: assets/javascripts/app/models.js (hunk 1 above)
Comment (seancribbs) [inline assets/javascripts/app/models.js]: Good idea!

**Pull request merged**